### PR TITLE
[Feature] Support positive weights in BCE.

### PR DESCRIPTION
**Pull request by Ezra-Yu**
## Motivation

Add pos_weight in BCE

## Modification

Add pos_weight in BCE

## BC-breaking (Optional)
no

## Use cases (Optional)

```
pos_weight = [0.1, 0.2]
loss_cfg = dict(
        type='CrossEntropyLoss',
        use_sigmoid=True,
        reduction='mean',
        loss_weight=1.0,
        pos_weight=pos_weight)
loss = build_loss(loss_cfg)
```

## Checklist

**Before PR**:

- [x] Pre-commit or other linting tools are used to fix the potential lint issues.
- [x] Bug fixes are fully covered by unit tests, the case that causes the bug should be added in the unit tests.
- [x] The modification is covered by complete unit tests. If not, please add more unit test to ensure the correctness.
- [x] The documentation has been modified accordingly, like docstring or example tutorials.

**After PR**:

- [x] If the modification has potential influence on downstream or other related projects, this PR should be tested with those projects, like MMDet or MMSeg.
- [x] CLA has been signed and all committers have signed the CLA in this PR.

### CODE REVIEW

File: mmcls/models/losses/cross_entropy_loss.py [hunk 1]
```diff
@@ -178,12 +191,19 @@ def forward(self,
         else:
             class_weight = None
 
+        # only BCE loss has pos_weight
+        if self.pos_weight is not None and self.use_sigmoid:
+            pos_weight = cls_score.new_tensor(self.pos_weight)
```
Comment (mzr1996) [inline mmcls/models/losses/cross_entropy_loss.py:196]: Since `pos_weight` is only used in BCE loss, just pass the parameter here, and other cross entropy function won't need to add this useless argument.
```suggestion
            pos_weight = cls_score.new_tensor(self.pos_weight)
            kwargs.update({'pos_weight': pos_weight})
```

**Review by mzr1996 — CHANGES_REQUESTED** [review]

File: mmcls/models/losses/cross_entropy_loss.py (hunk 1 above)
Comment (Ezra-Yu) [reply mmcls/models/losses/cross_entropy_loss.py:196]: fixed

**Review by mzr1996 — APPROVED** [review]
LGTM
